examples/specs/box_plot.json

```diff
@@ -0,0 +1,81 @@
+{
+  "description": "A error bar plot showing mean, min, and max in the US population distribution of age groups in 2000.",
+  "data": {"url": "data/population.json"},
+  "layers": [
+    {
+      "mark": "rule",
+      "encoding": {
+        "x": {"field": "age","type": "ordinal"},
+        "y": {
+          "aggregate": "min",
+          "field": "people",
+          "type": "quantitative",
+          "axis": {"title": "population"}
+        },
+        "y2": {
+          "aggregate": "max",
+          "field": "people",
+          "type": "quantitative",
+          "axis": {"title": "population"}
+        }
+      }
+    },
+    {
+      "mark": "tick",
+      "encoding": {
+        "x": {"field": "age","type": "ordinal"},
+        "y": {
+          "aggregate": "min",
+          "field": "people",
+          "type": "quantitative",
+          "axis": {"title": "population"}
```

Comment:
Agreed to only say `axis.title` once for layers. My vote would be for the first layer, such that it feels like it cascades down the textual specification (or layered up in the canvas).